.snyk

```diff
@@ -10,10 +10,10 @@ ignore:
     - '@azure/identity > keytar > prebuild-install > npmlog > gauge > strip-ansi > ansi-regex':
         reason: None given
         expires: '2022-06-17T09:40:34.194Z'
-      '@azure/identity > keytar > prebuild-install > npmlog > gauge > wide-align > string-width > strip-ansi > ansi-regex':
-        reason: None given
-        expires: '2022-06-17T09:44:39.068Z'
     - '@azure/identity > keytar > prebuild-install > npmlog > gauge > string-width > strip-ansi > ansi-regex':
         reason: None given
         expires: '2022-06-17T09:40:34.194Z'
+    - '@azure/identity > keytar > prebuild-install > npmlog > gauge > wide-align > string-width > strip-ansi > ansi-regex':
+        reason: None given
+        expires: '2022-06-17T09:44:39.068Z'
```

Comment:
I think we can fix this vulnerability by updating to version 2 of the identity package.  We're using it without issue in other services.